src/analysis/retail/deathknight/blood/modules/features/Ossuary.tsx

```diff
@@ -40,12 +46,7 @@ class Ossuary extends Analyzer {
     } else {
       this.dsWithoutOS += 1;
 
-      event.meta = event.meta || {};
-      event.meta.isInefficientCast = true;
-      event.meta.inefficientCastReason = defineMessage({
-        id: 'deathknight.blood.ossuary.ineffectiveCast',
-        message: `This Death Strike cast was without Ossuary.`,
-      });
+      addInefficientCastReason(event, ineffectiveCastMessage);
```

Comment:
this one will conflict with my blood apl pr because its actually broken and displays `[Object object]` right now :see_no_evil: will be an easy conflict to fix though. will revert the change in my pr because this solution is better.